clients/go/pkg/audit/interceptor.go

```diff
@@ -44,6 +44,29 @@ import (
 
 type auditLogReqKey struct{}
 
+const (
+	interceptorErrPrefix = "audit interceptor: "
+)
+
+// IsInterceptorErr returns true if the grpc status error is an Lumberjack
+// interceptor error.
+func IsInterceptorErr(err error) bool {
+	// Because these errors are grpc status error, we can't assert their error type.
+	// We can only check the error code and message.
+	if status.Code(err) != codes.Internal && status.Code(err) != codes.FailedPrecondition {
+		return false
+	}
+	return strings.Contains(err.Error(), interceptorErrPrefix)
```

Comment:
I think it'd be better if we unwrapped and checked for the prefix instead of a substring?

Reply:
Yes, switched to a better way.